docs/pages/modules/use-create-expo-module-in-your-project.mdx

```diff
@@ -0,0 +1,180 @@
+file in the app and render the `hello` text from our `expo-settings` module:
+
+```javascript App.js
+import * as Settings from 'expo-settings';
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export default function App() {
+  return (
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+      <Text>{Settings.hello()}</Text>
+      <StatusBar style="auto" />
+    </View>
+  );
+}
+```
+
+If you've configured everything correctly you should see a text saying "Hello world! 👋” in your app.
+
+## Publishing your module
+
+If you don't want to go through the process of setting up a monorepo, you can also publish your module to npm and just add it to your projects as a normal dependency.
+
+### 1. Initialize a new module
+
+First, start by creating a new module using `create-expo-module`. Pay attention to each prompt as we're going to publish this library, so make sure to choose a unique name for your npm package.
+
+<Terminal cmd={['$ npx create-expo-module expo-settings']} />
+
+### **2. Run the example project**
+
+Now let's run the example project to make sure everything is working. Start the TypeScript compiler to watch for changes and rebuild the module JavaScript.
+
+<Terminal
+  cmd={[
+    '# Run this in the root of the project to start the TypeScript compiler',
+    '$ npm run build',
+  ]}
+/>
+
+In another terminal window, compile and run the example app:
+
+<Terminal
+  cmdCopy="cd example && npx expo run:ios"
+  cmd={[
+    '$ cd example',
+    '# Run the example app on Android',
+    '$ npx expo run:android',
+    '# Run the example app on iOS',
+    '$ npx expo run:ios',
+  ]}
+/>
+
+### 3. Publish your package to npm
+
+In order to publish your package to npm, you need to have an npm account. If you don't have one, first register through [the npm website](https://www.npmjs.com/signup). Once you have an account, log in using the following command:
+
+<Terminal cmd={['$ npm login']} />
+
+Then, navigate to the root of your module and publish your module running:
+
+<Terminal cmd={['$ npm publish']} />
+
+Your module is now published to npm and can be installed in other projects using `npm install`.
+
+Apart from publishing your module to npm, there are other ways to use it in your project. One way is to create a tarball of your module using `npm pack` and install it directly in your project by running `npm install /path/to/tarball`. This is useful if you want to test your module locally before publishing it to npm or if you want to share it with others who don't have access to the npm registry. Another way is to run a local npm registry and install your module from there.
+
+If you want to publish a private private package or use a private registry with EAS Builds you can read how to configure it [here](https://docs.expo.dev/build-reference/private-npm-packages/).
```

Comment:
```suggestion
If you want to [publish a private package or use a private registry with EAS Builds](https://docs.expo.dev/build-reference/private-npm-packages/).
```

You can also add this as a bullet point to the above as I mentioned.

Reply:
updated